releasenotes/notes/add-integration-field-to-topology-link-metadata-8a35c0a70ac80b1e.yaml

```diff
@@ -0,0 +1,15 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Emit new field: integration for TopologyLinkMetadata for better metrics collection.
+issues:
+  - |
+    List known issues here, or remove this section.
```

Comment:
```suggestion
```

Reply:
woopsie ty!!

Reply:
also love the ur avatar hat (takoyaki??)